browser/ui/brave_browser_command_controller.cc

```diff
@@ -78,6 +80,11 @@ void BraveBrowserCommandController::InitBraveCommandState() {
 #endif
   UpdateCommandForBraveAdblock();
   UpdateCommandForTor();
+  const base::CommandLine& command_line =
```

Comment:
we need to wrap this check in a static method like `BraveSyncService::is_enabled()`